cantools/database/can/formats/arxml.py

```diff
@@ -492,9 +492,18 @@ def _load_signal_type(self, i_signal):
 
             base_type_encoding = base_type_encoding.text
 
-            if base_type_encoding == '2C':
+            if base_type_encoding in ('2C', '1C', 'SM'):
+                # types which use two-complement, one-complement or
+                # sign+magnitude encodings are signed. TODO (?): The
+                # fact that if anything other than two complement
+                # notation is used for negative numbers is not
+                # reflected anywhere. In practice this should not
+                # matter, though, since two-complement notation is
+                # basically always used for systems build after
+                # ~1970...
                 is_signed = True
             elif base_type_encoding == 'IEEE754':
+                is_signed = True # all standardized IEEE-754 floating point types are signed
```

Comment:
`is_signed` is only used for integers. It is never used in combination with `is_float`. At least that is how this package is designed atm.

Reply:
okay, thanks for the clarification; this wasn't clear to me. maybe `is_signed` should be renamed to `is_signed_int`?

Reply:
It's fine as is. Maybe clarify in its property docstring.

Reply:
okay, I'll remove the patch...

Reply:
done